signac/contrib/project.py

```diff
@@ -305,21 +301,26 @@ def doc(self):
     def doc(self, new_doc):
         self.document = new_doc
 
+    @property
+    def stores(self):
+        return H5StoreManager(self._rd)
+
     @property
     def data(self):
         """The data associated with this project.
 
+        Equivalent to:
+
+            return project.store['signac_data']
+
         :return: An HDF5-backed datastore.
-        :rtype: :class:`~signac.core.h5store.H5Store`
+        :rtype: :class:`~signac.core.h5store.H5DictManager`
```

Comment:
This is still returning an H5Store instance, right? In any case, H5DictManager should just be H5StoreManager.